controllers/auth.js

```diff
@@ -0,0 +1,59 @@
+const User = require("../models/User");
+const { StatusCodes } = require("http-status-codes");
+const { BadRequestError, UnauthenticatedError } = require("../errors");
+
+const register = async (req, res) => {
+  console.log("in auth.register:");
+  console.log("req.body:  ", req.body);
+
+  const user = await User.create({ ...req.body });
```

Comment:
Nitpick:
Since `req.body` is already an object, you won’t need to create a new object and spread out the body inside of it:
```diff
- const user = await User.create({ ...req.body });
+ const user = await User.create(req.body);
```